src/wallet/wallet.h

```diff
@@ -1092,7 +1092,7 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
     bool DummySignTx(CMutableTransaction &txNew, const std::vector<CTxOut> &txouts, bool use_max_sig = false) const;
     bool DummySignInput(CTxIn &tx_in, const CTxOut &txout, bool use_max_sig = false) const;
 
-    bool ImportScripts(const std::set<CScript> scripts) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
+    bool ImportScripts(const std::set<CScript> scripts, int64_t time = -1) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
```

Comment:
In commit "Optionally allow ImportScripts to set script creation timestamp" (92863d7de8cf0d7432bc47f36bf1aa4de257c548)

I think it'd be better not to have a default parameter so callers will have a prompt to provide this information if it's available and not just drop it in the floor. There shouldn't be a lot of preexisting calls.

Reply:
Done